README.md

```diff
@@ -1,36 +1,60 @@
 # Trustlines Merkle-Drop
 
-Example of usage to compute the merkle root from a json file containing certain addresses and balance:
-```
-merkle-drop root airdrop.csv
+## Installation
+
+Please make sure you have the following requirements installed:
+
+- solidity compiler >= v0.5.8
+- python >= 3.6
+
+Activate a fresh virtualenv and run `make install`. This will install
+the `merkle-drop` program. Please run `merkle-drop --help` to get an
+overview of available command line options. This document covers only
+a subset of the available options.
+
+
+## Compute the merkle root
+
+The `root` subcommand computes the merkle root from a CSV file
+containing addresses and balances.
+
+```shell
+$ merkle-drop root airdrop.csv
+0x5a7973598b5c9040bb842e4ff16c4685b4fd4bd1ac62645b9b844a9df35592d6
 ```
 
 Example for `airdrop.csv`:
-```
+```CSV
```

Comment:
We had this confusion already once: can we please add a comment here that the amount depends on the decimal count of the token to drop? We typically use 18 decimals. So we could adjust the example to have values like `10000000000000000000` which would refer to `10` "full" token units.